test/e2e_node/dra_test.go

```diff
@@ -78,6 +78,10 @@ const (
 	// Also used as timeout in other tests because it's a good upper bound
 	// even when the test normally completes faster.
 	retryTestTimeout = kubeletRetryPeriod + 30*time.Second
+
+	// pluginMonitorTimeout is the duration after which Kubelet closes the connection to the plugin
+	// if it doesn't respond to GetInfo GRPC calls.
+	pluginMonitorTimeout = time.Minute
```

Comment:
Can this be a constant exported by a kubelet API package? Otherwise this might get out of sync with the implementation.

Reply:
This would be an awkward expression, which would be hard to understand. Something like this: dialTimeout + maxFailures * connectionCheckInterval + <some delay on top of it to ensure the test waited enough>

Reply:
If it's an awkward expression, then how can the test be sure that one minute is that duration? Or is the an estimate?

Reply:
It's an estimate based on that awkward expression and then proven by running the test many times.

Reply:
Then let's document it like that:

*"pluginMonitorTimeout is the duration after which Kubelet is expected to have closed the connection to the plugin if it doesn't get a response to the GetInfo gRPC calls. This is an upper bound, typically the kubelet is faster."*

Reply:
Thank you for the suggestion! Done.